src/Exceptionless.Core/Plugins/WebHook/Default/005_SlackPlugin.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Exceptionless.Core.Pipeline;
+using Exceptionless.Core.Plugins.Formatting;
+
+namespace Exceptionless.Core.Plugins.WebHook;
+
+[Priority(5)]
+public sealed class SlackPlugin : WebHookDataPluginBase {
+    private readonly FormattingPluginManager _pluginManager;
+
+    public SlackPlugin(FormattingPluginManager pluginManager, AppOptions options) : base(options) {
+        _pluginManager = pluginManager;
+    }
+
+    public override Task<object> CreateFromEventAsync(WebHookDataContext ctx) {
+        if (String.IsNullOrEmpty(ctx.WebHook.Url) || !ctx.WebHook.Url.EndsWith("/slack"))
```

Comment:
I'm confused, if I go to discord and tell it to create a webhook URL for me, I don't get to pick the URL. So how can I make it end with `/slack`? And if the purpose of this PR is to trigger the webhook data to be formatted a certain way then I'd rather see us do that with a query string parameter. Also, we should consider automatically doing it for known endpoints like `https://discord.com/api/webhooks/`. If the URL starts with that, then we know it's a discord webhook so we could automatically trigger it to send the payload in a format that discord understands.

Reply:
Also, I'm guessing that discord supports the slack message format which is why we are doing this?